example/CMakeLists.txt

```diff
@@ -37,18 +37,7 @@ if(${ISAAC_STEREO} STREQUAL "Anaglyph")
 endif()
 
 
-if (ISAAC_ALPAKA)
-	ALPAKA_ADD_EXECUTABLE(isaac_example_alpaka ${CMAKE_SOURCE_DIR}/example.cpp)
-	target_link_libraries(isaac_example_alpaka PUBLIC ${ISAAC_LIBRARIES})
-	install(TARGETS isaac_example_alpaka DESTINATION bin )
-endif()
-
-if (ISAAC_CUDA)
-	REMOVE_DEFINITIONS(REMOVE_ITEM "-DISAAC_ALPAKA")
-	set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS}" -std=c++11)
-    list(REMOVE_DUPLICATES CUDA_NVCC_FLAGS)
-	SET_SOURCE_FILES_PROPERTIES(${CMAKE_SOURCE_DIR}/example.cpp PROPERTIES CUDA_SOURCE_PROPERTY_FORMAT OBJ)
-	CUDA_ADD_EXECUTABLE(isaac_example_cuda ${CMAKE_SOURCE_DIR}/example.cpp)
-	target_link_libraries(isaac_example_cuda ${ISAAC_LIBRARIES})
-	install(TARGETS isaac_example_cuda DESTINATION bin )
-endif()
+ALPAKA_ADD_EXECUTABLE(isaac_example ${CMAKE_SOURCE_DIR}/example.cpp)
+set(ISAAC_LIBRARIES ${ISAAC_LIBRARIES} "alpaka::alpaka")
```

Comment:
at @psychocoderHPC: 
@FelixTUD mentioned that here alpaka probably changed with the last version tested - can you confirm this?

Reply:
This look valid and should work. @FelixTUD do you tested it with PIConGPU dev?

Reply:
There is no Problem with PIConGPU dev, only with the standalone example, currently I added it only to the example and I think it should replace the 4 alpaka defines discussed below in the lib/ISAACConfig.cmake, right?